modules/fluid_properties/doc/content/syntax/FluidProperties/index.md

```diff
@@ -0,0 +1 @@
+!template load file=stubs/moose_system.md.template name=FluidProperties syntax=/FluidProperties
```

Comment:
What's this doing here?

Reply:
That's garbage from a previous run to generate .md stubs. Thanks for catching that...